crates/stackable-operator/src/builder/pod/container.rs

```diff
@@ -490,20 +539,18 @@ mod tests {
             "lengthexceededlengthexceededlengthexceededlengthexceededlengthex";
         assert_eq!(long_container_name.len(), 64); // 63 characters is the limit for container names
         let result = ContainerBuilder::new(long_container_name);
-        match result
+        if let Error::InvalidContainerName {
+            container_name,
+            source,
+        } = result
             .err()
             .expect("Container name exceeding 63 characters should cause an error")
         {
-            Error::InvalidContainerName {
-                container_name,
-                source,
-            } => {
-                assert_eq!(container_name, long_container_name);
-                assert_eq!(
-                    source.to_string(),
-                    "input is 64 bytes long but must be no more than 63"
-                )
-            }
+            assert_eq!(container_name, long_container_name);
+            assert_eq!(
+                source.to_string(),
+                "input is 64 bytes long but must be no more than 63"
+            )
```

Comment:
thought (non-blocker): I'm not sure how I feel about asserting error message strings.